client/src/curriculum/landing.tsx

```diff
@@ -135,7 +136,12 @@ function About({ section }) {
         <DisplayH2 id={id} title={title} />
         <div className="about-content" dangerouslySetInnerHTML={html}></div>
         <div className="arrow"></div>
-        <ScrimIframe url={SCRIM_URL}>
+        <section className="scrim-wrapper">
+          <div className="scrim-border">
+            <Suspense fallback={<scrim-inline />}>
```

Comment:
if threre's an easy way we could wrap `ScrimInline` programmatically in `scrim-inline.ts`

Reply:
It's a little bit of a chicken and egg problem in that we need to load `scrim-inline.ts` async in order for it to not bloat our bundle size, but the `Suspense` stuff all needs to be loaded sync for it to actually work: so the helper function would need to live in another file. If this becomes a common pattern for us, I think we can do that then.